base/src/main/proto/spine/options.proto

```diff
@@ -232,7 +232,54 @@ extend google.protobuf.FieldOptions {
     //
     bool beta = 73853;
 
-    // Reserved 73854 to 73899 for future options.
+    // Marks an entity state field as column.
+    //
+    // The column fields are stored separately from the entity record and can be specified as
+    // filtering criteria during entity querying.
+    //
+    // The value of a column field can be set in two ways:
+    //
+    // 1. In the handler methods of the entity, just like any other part of entity state.
+    // 2. Using the generated `_X_WithColumns` interface. For example:
+    //
+    // // Assuming a message storing entity state.
+    // message UserProfile {
+    //    ...
+    //    int32 year_of_registration = 8 [(column) = true];
+    // }
+    //
+    // // A special interface will be generated by Spine Model Compiler.
+    // interface UserProfileWithColumns extends EntityWithColumns {
+    //
+    //    int getYearOfRegistration();
+    // }
+    //
+    // // An entity class which implements the `UserProfileWithColumns` interface.
+    // class UserProfileProjection
+    //            extends Projection<UserId, UserProfile, UserProfile.Builder>
+    //            implements UserProfileWithColumns {
+    //      ...
+    //
+    //      @Override
```

Comment:
I am a bit confused by the Java examples in the language-agnostic Protobuf definition of the column.

Can we please think of some other place to list the Java examples and leave just the generic definition of what a column is. Think of this Protobuf file as of such used by JavaScript and C++ clients as well.